django/contrib/gis/db/models/manager.py

```diff
@@ -9,3 +12,11 @@ class GeoManager(Manager.from_queryset(GeoQuerySet)):
     # so that geometry columns on Oracle and MySQL are selected
     # properly.
     use_for_related_fields = True
+
+    def __init__(self, *args, **kwargs):
+        warnings.warn(
+            "The GeoManager class is deprecated. Simply use a normal manager "
+            "once all legacy GeoQuerySet methods have been replaced by annotations.",
```

Comment:
Maybe use the 2nd person? .e.g "once you have replaced all calls to GeoQuerySet methods by annotations"? Right now it's not obvious to me if the legacy methods will be replaced by Django or if it's something that I need to do.